src/components/table/use-cases/generations.tsx

```diff
@@ -237,10 +263,54 @@ export default function GenerationsTable({ projectId }: GenerationsTableProps) {
       enableSorting: true,
     },
     {
-      accessorKey: "cost",
-      header: "Cost",
+      accessorKey: "latencyPerToken",
+      id: "latencyPerToken",
+      header: "Latency per Token",
+      cell: ({ row }) => {
+        const latency: number | undefined = row.getValue("latency");
```

Comment:
I think this was a bit broken already: we cast the Decimals to numbers here and below. I think we should handle these gracefully.